projects/knora-ui/src/lib/viewer/viewer.module.ts

```diff
@@ -5,9 +5,10 @@ import {MatInputModule} from '@angular/material';
 import {CommonModule} from '@angular/common';
 import { IntValueComponent } from './values/int-value/int-value.component';
 import { DisplayEditComponent } from './operations/display-edit/display-edit.component';
+import { DecimalValueComponent } from './values/decimal-value/decimal-value.component';
 
 @NgModule({
-  declarations: [TextValueAsStringComponent, IntValueComponent, DisplayEditComponent],
+  declarations: [TextValueAsStringComponent, IntValueComponent, DisplayEditComponent, DecimalValueComponent],
   imports: [
```

Comment:
Please also add `DecimalValueComponent` to exports, otherwise it cannot be used standalone.